src/main/java/org/snomed/aag/data/services/ProjectAcceptanceCriteriaService.java

```diff
@@ -162,6 +164,12 @@ public ProjectAcceptanceCriteria findByBranchPathWithRelevantCriteriaItems(Strin
         // Remove from collection if Criteria is no longer relevant in comparison to Branch metadata.
         relevantCriteriaItems.removeIf(isConflictBetweenAuthorFlags(branchAuthorFlags, authorFlagsAll, authorFlagsEnabled));
 
+        // Add to collection if task Branch received content via batch
+        if (criteria.isBranchTaskLevel(branchPath) && authorFlagsEnabled.contains(Constants.AUTHOR_FLAG_BATCH_CHANGE)) {
+            criteria.setBatch(true);
+            addRelevantCriteriaItemsForBatchTask(relevantCriteriaItems);
+        }
+
```

Comment:
If the branch is a task branch, and the branch itself has the appropriate flag, the TASK_VALIDATION_CLEAN CriteriaItem is added to the collection (if it is not already there).